app/helpers/rails_admin/application_helper.rb

```diff
@@ -26,11 +26,11 @@ def edit_user_link
       content = [
         RailsAdmin::Config.show_gravatar && _current_user.email.present? && image_tag("#{request.ssl? ? 'https://secure' : 'http://www'}.gravatar.com/avatar/#{Digest::MD5.hexdigest _current_user.email}?s=30", alt: ''),
         content_tag(:span, _current_user.email),
-      ].compact.join.html_safe
+      ].map(&:presence).compact.join.html_safe
```

Comment:
```suggestion
      ].filter(&:present?).join.html_safe
```
will be a bit concise.